src/components/CustomerCard.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
```

Comment:
I think this is the outdated customer card. The newer one got merged right before bug bash. Can you pull from main?